state/memory.go

```diff
@@ -420,7 +420,7 @@ func collectLogs(s *snapshot) []*types.Log {
 	return logs
 }
 
-func (db *inMemoryStateDB) GetLogs(txHash common.Hash, block uint64, blockHash common.Hash) []*types.Log {
+func (db *inMemoryStateDB) GetLogs(txHash common.Hash, block uint64, blockHash common.Hash, blkTimestamp uint64) []*types.Log {
```

Comment:
`collectLogs` should probably also use the block timestamp to produce results equal to geth.